src/windows/process.rs

```diff
@@ -527,6 +512,146 @@ unsafe fn get_cmdline_from_buffer(buffer: *const u16) -> Vec<String> {
     res
 }
 
+#[cfg(target_pointer_width = "64")]
```

Comment:
Instead of having two functions, could you instead do:

```rust
if !cfg!(target_pointer_width = "64") {{
    return;
}
```

Like that no need to have cfg on imports.

Reply:
Done